packages/text-editor/src/components/TextEditor.svelte

```diff
@@ -62,8 +62,11 @@
     }
   }
   export function submit (): void {
-    content = editor.getHTML()
-    dispatch('content', content)
+    const content = editor.getHTML().replace(/<[^>]*>/g, '');
```

Comment:
I think this check is inaccurate, there can be elements that don't have inner content, such as images. For such content the content will be empty.